### ISSUE TITLE
`golemsp settings show` shows different result in subsequent runs

### ISSUE THREAD

**Issue by johny-b** [golemfactory/yagna#1321]
Nothing was changed between first and second `golemsp settings show`:
```
$ golemsp settings show
node name: "jb-t1"
Shared resources:
        cores:  11
        memory: 21.317225322127342 GiB
        disk:   157.57481689453127 GiB


Pricing:

            0 GLM for start
         0.02 GLM per hour
          0.1 GLM per cpu hour
$ golemsp settings show
node name: "jb-t1"
Shared resources:
        cores:  11
        memory: 21.317225322127342 GiB
        disk:   157.57481689453127 GiB


Pricing:

            1 GLM for start
          360 GLM per hour
         3600 GLM per cpu hour
 ```
  
 This effect disappeared when I changed all `usage-coeffs` in my `.local/share/ya-provider/presets.json` to the same values.

**Comment by jiivan** [comment]
`golemsp` only shows single preset values. It is consistent with the way the installer works (by only letting user set the same values for all presets).

Not sure how to approach this. Not sure if showing all presets would be better than current solution which works for a typical user.

**Comment by Wiezzel** [comment]
If proper handling of multiple presets is too much work for now, maybe consider displaying some error/warning in case there is more than one preset.

This might be important because Erigolem setup relies on adding a new preset. The same goes with any custom runtime.

**Comment by jiivan** [comment]
fixed by: #1575
